web/src/Setting.js

```diff
@@ -801,6 +802,13 @@ export function getProviderSubTypeOptions(category, type) {
     } else {
       return [];
     }
+  } else if (type === "GitHub") {
+    return (
+      [
+        {id: "gpt-4o", name: "GPT-4o"},
+        {id: "gpt-4o-mini", name: "GPT-4o-mini"},

```

Comment:
Not only two models?

Reply:
A new commit has been pushed. GitHub offers over 40 models, and I have excluded some older ones and a few others that are not quite suitable.